resources/prosody-plugins/token/util.lib.lua

```diff
@@ -177,6 +185,39 @@ local function _verify_token(session, token)
     end
 end
 
+local function _verify_room_and_domain(room_address_to_verify, room, domain)
+
+    local room_to_verify,_,_ = jid.split(room_address_to_verify);
+    if not enableDomainVerification then
+        -- extract room name using all chars, except the not allowed ones
+
+        if room_to_verify == nil then
+            log("error",
+                "Unable to get name of the MUC room ? to: %s", room_address_to_verify);
+            return false;
+        end
+        return room_to_verify == string.lower(room);
+    end
+
+    -- make sure we remove any resource when comparing
+    room_address_to_verify = jid.bare(room_address_to_verify);
+    local target_subdomain, target_room = room_to_verify:match("^%[([^%]]+)%](.+)$");
```

Comment:
what is the format of this multidomain thing ? could you please add a comment with example ?